### PR TITLE
add less compilation into gulp

### PR DESCRIPTION
**Pull request by ChengshuLi**

### CODE REVIEW

File: codalab/apps/web/gulpfile.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 var gulp = require('gulp');
 var babel = require('gulp-babel');
 var del = require('del');
+var less = require('gulp-less');
+var path = require('path');
 
 // Compile JSX for production
 gulp.task('default', function() {
```
Comment (percyliang) [inline codalab/apps/web/gulpfile.js]: make this jsx, so that it's symmetric with less

File: codalab/apps/web/gulpfile.js [hunk 2]
```diff
@@ -15,3 +17,11 @@ gulp.task('default', function() {
 gulp.task('clean', function() {
     del(['static/dist']);
```
Comment (percyliang) [inline codalab/apps/web/gulpfile.js]: clean up less too

File: codalab/apps/web/package.json [hunk 3]
```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "gulp && gulp less",
```
Comment (percyliang) [inline codalab/apps/web/package.json]: should be gulp jsx && gulp less

File: codalab/apps/web/package.json [hunk 4]
```diff
@@ -5,7 +5,9 @@
...
     "jsx": "gulp",
```
Comment (percyliang) [inline codalab/apps/web/package.json]: gulp jsx

**Comment by kashizui** [comment]
@ChengshuLi @percyliang should we also stop tracking the generated CSS files in the repo?

**Comment by kashizui** [comment]
Fixes #83

**Pull request merged**